acceptance/setup/aio/pre-suite/015_PackageHostsPresets.rb

```diff
@@ -1 +1,9 @@
-master['use-service'] = true
+if options['is_puppetserver']
+  master['use-service'] = true
+  master['puppetservice'] = 'puppetserver'
+  master['puppetserver-confdir'] = '/etc/puppetlabs/puppetserver/conf.d'
```

Comment:
do these need to be set in the master hash?  can't we get them from the options hash?